systers_portal/templates/base.html

```diff
@@ -165,6 +165,7 @@
 
 <script src="{% static 'js/libs/jquery-1.11.1.min.js' %}"></script>
 <script src="{% static 'js/libs/bootstrap.min.js' %}"></script>
+<script src="{% static 'js/edit_profile.js' %}"></script>
```

Comment:
The file name can be renamed to `profile_choose_file.js` or a better name letting the developers know that the file is for the Choose file button in the edit profile form.